reporter/src/main/kotlin/reporters/ExcelReporter.kt

```diff
@@ -113,7 +113,7 @@ class ExcelReporter : Reporter {
         val workbook = XSSFWorkbook()
 
         defaultStyle = workbook.createCellStyle().apply {
-            setVerticalAlignment(VerticalAlignment.TOP)
+            verticalAlignment = VerticalAlignment.TOP
```

Comment:
Actually, I believe the inspection is wrong here, as I recall this did not compile. Let's see what the CI checks say.

Reply:
I remember the same, but it was a long time and several Kotlin versions ago, maybe something was improved in the meantime.